src/ubuntu/18.04/webassembly/Dockerfile

```diff
@@ -24,3 +24,13 @@ RUN git clone https://github.com/emscripten-core/emsdk.git ${EMSDK_PATH} \
     && ./emsdk install ${EMSCRIPTEN_VERSION}-upstream \
     && ./emsdk activate ${EMSCRIPTEN_VERSION}-upstream \
     && chmod -R 777 ${EMSDK_PATH}
+
+# Install V8 Engine to be able to execute Trimming Tests
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - \
+    && apt install -y nodejs \
```

Comment:
Installing nodejs shouldn't be necessary.  It's already installed in the base image at https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/f339b3b50c34cf1e4116ea7a94f25369debaee0c/src/ubuntu/18.04/coredeps/Dockerfile#L7

Reply:
True but for v8 engine we will need npm 10+ which will only be available after the previouse curl command. After running that, the `apt install nodejs` will install the new npm which will be able to install the v8 engine. With the nodejs that is preinstalled in the container we can't install v8.

Reply:
@joperezr we don't need to install v8 from npm/jsvu, for the helix queue we just download the binary from blob storage: https://dev.azure.com/dnceng/internal/_git/dotnet-helix-machines?path=%2Fartifacts%2Flinux-v8-engine%2Flinux-v8-engine.sh&_a=contents&version=GBmaster